apps/public-docsite/src/pages/Overviews/ControlsPage/ControlsPage.tsx

```diff
@@ -55,14 +55,16 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
               </ul>
             ),
           },
-      );
+      ) as IPageSectionProps<Platforms>[];
 
-    _otherControlsRequestSections(platform) !== undefined && sections.push(_otherControlsRequestSections(platform));
+    const _otherControlsRequestSectionsValue = _otherControlsRequestSections(platform);
```

Comment:
typescript cannot narrow down nullables via shortcircuiting - separate assignement needs to be created and used in conditions.